metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/database/SraObjectsImporterThroughDatabase.java

```diff
@@ -200,21 +141,6 @@ protected Study extractAnalysisFromStudy(StudyType studyType, Study study) {
      *
      * @return combined accession and refname. Example: "GCA_000001405.1#GRCh37"
      */
-    @Override
```

Comment:
Comments haven't been removed.

Reply:
Whoops, fixed in https://github.com/EBIvariation/metadata-ws/pull/58/commits/24cc13de55fe41b8224bf3eb6eb0facf6f6e3d92